R/mergeReport.r

```diff
@@ -322,3 +341,54 @@ getSurfaces.default <- function(x, ...){
     stop("Please provide a named list that includes a surfaces matrix")
   }
 }
+
+#' Number of discharge values
+#' 
+#' Determine the number of discharge data points in an eList
+#' 
+#' @param eList named list with at least Daily dataframe
```

Comment:
If you want to mess around, you could make a single @rdname called "Helper functions" or something like that, so they only get 1 total help file for all 3 functions. We could add print.egret and maybe all of the getInfo, getDaily, etc too.